android/src/main/java/com/swmansion/reanimated/nodes/FormatNode.java

```diff
@@ -0,0 +1,31 @@
+package com.swmansion.reanimated.nodes;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.swmansion.reanimated.NodesManager;
+import com.swmansion.reanimated.Utils;
+import java.text.NumberFormat;
+import java.text.DecimalFormat;
+
+public class FormatNode extends Node {
+  private NumberFormat nf;
```

Comment:
can we stick to Android java naming convention (as it is being used elsewhere) here and name it `mNumberFormat` or sth that starts from lowercase `m`